src/components/Pagination/PageButton/PageButton.js

```diff
@@ -1,26 +1,53 @@
 import styled, { css } from 'react-emotion';
 
 import Button from '../../Button';
+import { calculatePadding } from '../../Button/components/RegularButton/RegularButton'; // eslint-disable-line max-len
 
 const baseStyles = ({ theme }) => css`
   label: pagination__buttom;
   border-radius: 0;
   border-right: 0;
+  font-weight: normal;
   border-color: ${theme.colors.n300};
 
   &:focus {
-    border-right: ${theme.borderWidth.mega} solid ${theme.colors.n500};
+    border-color: ${theme.colors.n300};
+    border-width: ${theme.borderWidth.kilo};
+    padding: ${calculatePadding({ theme, size: 'kilo' })()};
   }
 
   &:hover {
-    border-right: ${theme.borderWidth.kilo} solid ${theme.colors.n500};
+    box-shadow: 1px 0 0 ${theme.colors.n500};
+    z-index: 1;
+  }
+  &:hover:focus {
+    box-shadow: none;
```

Comment:
Why are you removing the `box-shadow` here?

Reply:
Because otherwise it leaves a border where there should be none.

Reply:
Ah, I didn't see that the hover and focus states use different border-styles. 👍